### PR TITLE
Enabled sync of messages from kafka to CB for CDC audit

### PR DESCRIPTION
**Pull request by Shukla-Ankur**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+env
+connect_standalone_config="/app/kafka-connect-couchbase/config/connect-standalone.properties"
+couchbase_sink_config="/app/kafka-connect-couchbase/config/quickstart-couchbase-sink.properties"
+
+TICKLEDB_AUDIT_KAFKA_BOOTSTRAP_SERVER="10.0.4.125:9092"
+TICKLEDB_AUDIT_CB_CLUSTER_IP="10.0.3.215"
```
Comment (sumitjhaMindtickle) [inline docker-entrypoint.sh:10]: remove hard coded IPs

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export KAFKA_HOME=/app/kafka_2.12-2.2.0
+export KAFKA_CONNECT_COUCHBASE_HOME=/app/kafka-connect-couchbase
+cd /app/kafka-connect-couchbase
+mvn package
+unzip /app/kafka-connect-couchbase/target/kafka-connect-couchbase-3.4.6-SNAPSHOT.zip -d /app/kafka-connect-couchbase/target/
+
+cp ./config/connect-standalone.properties $KAFKA_HOME/config
+cp ./config/quickstart-couchbase-sink.properties $KAFKA_CONNECT_COUCHBASE_HOME/target/kafka-connect-couchbase-3.4.6-SNAPSHOT/config/
+
+KAFKA_HEAP_OPTS="-Xms1g -Xmx2g"
```
Comment (sumitjhaMindtickle) [inline docker-entrypoint.sh:34]: will it require 2 GB memory ?

Comment (sumitjhaMindtickle) [reply docker-entrypoint.sh:34]: To avoid fragmentation  , we generally avoid min and max value of heap allocated , lets make both the value same .

**Review by sumitjhaMindtickle — APPROVED** [review]

File: docker-entrypoint.sh (hunk 1 above)
Comment (Shukla-Ankur) [reply docker-entrypoint.sh:10]: These are default values, so keeping these

File: docker-entrypoint.sh (hunk 2 above)
Comment (Shukla-Ankur) [reply docker-entrypoint.sh:34]: Made these both same = 1GB

**Pull request merged**